### PR TITLE
Set log source default level to TRACE.

### PR DESCRIPTION
**Pull request by joaonc**
Addresses issue #1199 

Having the whole DOM in these keywords at the INFO level is too much clutter and causes the size of the logs to increase dramatically.

### CODE REVIEW

**Comment by joaonc** [comment]
@aaltat / @pekkaklarck The tests in this PR are failing ([here](https://travis-ci.org/robotframework/SeleniumLibrary/builds/460026885)), but the failures seem unrelated to the PR.. maybe the tests need to be updated?

**Comment by aaltat** [comment]
SeleniumLibrary uses also https://github.com/robotframework/statuschecker/ to verify log messages and keyword status from the output.xml. Also some test may initially fail, but are marked as passed with the status checker and vice versa. Based on the Travis it looks like a some of log check fails. Run the tests with the acceptance test runner: ` python atest/run.py <browser>` and fix the tests.

**Comment by aaltat** [comment]
Noticed few bugs in the https://github.com/robotframework/SeleniumLibrary/blob/master/atest/README.rst but now it should be fixed.

**Comment by joaonc** [comment]
Trying to verify that the DOM is _not_ added to the log at `INFO` level, but bumping into [this issue](https://github.com/robotframework/statuschecker/issues/4) with `robotstatuschecker`.

Can you think of another way I can check that the DOM is not **logged** in that step?

The other way I can think is to simply remove the check, but I'd rather check that is not there than not check at all.

**Comment by aaltat** [comment]
There is suggestion, in the https://github.com/robotframework/statuschecker/issues/4 how to overcome the problem. Would you need more help?

**Comment by joaonc** [comment]
Need some help.  The new test cases are passing, except [this one](https://github.com/robotframework/SeleniumLibrary/blob/5b96c804bca76dc8cf235ecac896f8c1b8e3f75e/atest/acceptance/keywords/content_assertions.robot#L70) (`Page Should Contain With Custom Log Level TRACE`) b/c loglevel TRACE is not working.

Even if I just do `Log    Whatever    TRACE` it won't log anything.

Any idea what's going on?
Followed the code and seemed ok.. TRACE should work.
Note that the previous test case (same but at DEBUG level) works as expected.

**Comment by pekkaklarck** [comment]
Without looking anything, my guess is that tests are run using the debug level and thus trace messages are ignored. You can use `Set Log Level` keyword to set it to trace in setup and again back to debug in teardown.

**Comment by joaonc** [comment]
Yes, that was it tx!
Tests are now passing except one job [here](https://travis-ci.org/robotframework/SeleniumLibrary/builds/461540244?utm_source=github_status&utm_medium=notification).  It's weird b/c I run w/ Python 3.6 on my local and works fine.

I wonder if it's b/c it's running on [Jython](https://travis-ci.org/robotframework/SeleniumLibrary/jobs/461540248#L473) in the test case [Create Webdriver Creates Functioning WebDriver](https://travis-ci.org/robotframework/SeleniumLibrary/jobs/461540248#L663), which should pass and doesn't seem related to my changes.

**Comment by aaltat** [comment]
Jython fails randomly in Travis, it is annoying but I have not been able to reproduce the problem locally. I did re-trigger the Jython run and lets see how it goes.

**Comment by joaonc** [comment]
All is green now, thanks!  Hope this gets merged for the next SeleniumLibrary release.

File: src/SeleniumLibrary/keywords/browsermanagement.py [hunk 1]
```diff
@@ -292,12 +292,12 @@ def log_location(self):
         return url
 
     @keyword
-    def log_source(self, loglevel='INFO'):
+    def log_source(self, loglevel='TRACE'):
```
Comment (aaltat) [inline src/SeleniumLibrary/keywords/browsermanagement.py]: Log source `loglevel` should not be changed.

**Review by aaltat — CHANGES_REQUESTED** [review]
It looks like you changed all the keywords log level. At least the `Log Source` log level should not have been changed. Also I did not prepare this big change and I will think it over.

**Comment by joaonc** [comment]
@aaltat made the fix you requested and now the Jython test is failing again..  can you kick it off again?


> did not prepare this big change and I will think it over

Note that it's the same change everywhere and works as discussed in issue #1199 
Also, added test cases to make sure the new logging level is as expected.

Sure hope this makes it..  the amount of unnecessary logging at `INFO` level is causing me to have to log at `ERROR` (only place where the DOM is not logged), which in turn causes me to miss out on important info and have to re-run tests when they fail  😞

File: src/SeleniumLibrary/keywords/browsermanagement.py (hunk 1 above)
Comment (joaonc) [reply src/SeleniumLibrary/keywords/browsermanagement.py]: Change added.

**Review by aaltat — APPROVED** [review]

**Comment by aaltat** [comment]
I think this is good to go. I need to create new issue, to that it matches to the scope of the PR.

**Pull request merged**